### DISCUSSION TITLE
Is It Possible To Create a "PartialControl" Type For a Controller With a Field Value That Is Shared Between Different Controls?

### DISCUSSION THREAD

**Discussion by dudiharush** [react-hook-form/react-hook-form#4702] [General]
Hi,
I have a  Controller, and I need it to get a control prop that has a filed value that is shared between two different Controls sub field values, is it possible?

ex:
 I manage cat and dog in different forms, and I want to share a component to manage their owners list.
I have `const catControl: Control<Cat> and const dogControl: Control<Dog>`
both Cat and Dog have prop: owners: string[], which is a list of their owners, and I want to manage this list of owners in a shared Controller component. How can I do it? My current "solution" is using `Control<any>` type, so i can pass both `Control<Cat>` and `Control<Dog>`

**Comment by bluebill1049** [comment]
can you share a csb?
